domains/lunareclipsestudio.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TBNRDev",
+        "email": "svenramirez28@gmail.com"
+    },
+    "record": {
+        "CNAME": "lunareclipsestudio.github.io"
```

Comment:
Redirect loop.

Reply:
Hi, thanks for the reply. can you tell me how to host the website in github pages

Reply:
https://docs.github.com/en/pages/quickstart